client/app/widgets/ToggleLocale/__tests__/ToggleLocale.spec.jsx

```diff
@@ -33,6 +33,7 @@ describe('ToggleLocale', () => {
         currentTarget: { value: 'es' },
       });
     });
+
```

Comment:
Since you're not working on migrating this test file, I would not include this in your PR.